src/02-data-types.js

```diff
@@ -79,7 +89,9 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  return text.includes(character);
```

Comment:
If your value is _already_ the `true` or `false` you need, you can simply return that value, as you did here with `.includes`. There's no need to tack on a `? true : false`, and you wisely saw that you could skip that.

See the solution below where you use an unnecessary ternary—`num % 2 === 0`, for example, is already the boolean you want. Apply what you did here to those problems!